projects/elife.yaml

```diff
@@ -2010,15 +2010,15 @@ fastly-logs:
                 project: elife-fastly
                 tables:
                     api_gateway:
-                        schema: ./src/buildercore/bigquery/schemas/fastly-logs-201810.json
+                        schema: bigquery/schemas/fastly-logs-201810.json
                     generic_cdn:
-                        schema: ./src/buildercore/bigquery/schemas/fastly-logs-201810.json
+                        schema: bigquery/schemas/fastly-logs-201810.json
                     journal:
-                        schema: ./src/buildercore/bigquery/schemas/fastly-logs-201810.json
+                        schema: bigquery/schemas/fastly-logs-201810.json
                     iiif:
-                        schema: ./src/buildercore/bigquery/schemas/fastly-logs-201810.json
+                        schema: bigquery/schemas/fastly-logs-201810.json
                     reproducible_document_stack:
-                        schema: ./src/buildercore/bigquery/schemas/fastly-logs-201810.json
+                        schema: bigquery/schemas/fastly-logs-201810.json
```

Comment:
this is too magic for me (as we have just removed various merging/overriding strategies that got out of hand), if it was `projects/data-pipeline/bigquery/schemas/...` it would require no additional code and would make the location clear?

Reply:
you have a low threshold for magic. I'll revert the paths